### ISSUE TITLE
Magento patch SUPEE-8788 will break the extension

### ISSUE THREAD

**Issue by tig-robertgrundeken** [Agence-DnD/PIMGento#176]
This important patch everybody must apply but breakes the pimgento extension. See problems occurs as the Magento 1.9.3 issue: https://github.com/Agence-DnD/PIMGento/issues/175

Pimgento now relies on functionality which is deprecated or dropped.

When will Pimgento come with a fix?

thx

**Comment by a2xchip** [comment]
@RobertGrundeken Hello! Can you provide more info, as we are about to patch our mage installation soon too

**Comment by RobertGrundeken** [comment]
What i see it breaks the menu option "**Catalog > Import Pim data**" for manually importing. As far as i can see the crons are working still ok. Nevertheless it is hard to explain to the users that there is an menu option resulting in a white page..

What is wrong:
With the patch SUPEE-8788 the file app/code/core/Mage/Adminhtml/Block/Media/Uploader.php is changed. Methods have been removed like getConfig() etc.
The file Pimgento /app/code/community/Pimgento/Core/Block/Adminhtml/Launcher.php is depending on this.

```
class Pimgento_Core_Block_Adminhtml_Launcher extends Mage_Adminhtml_Block_Media_Uploader
{

    /**
     * Constructor
     */
    public function __construct()
    {
        parent::__construct();

        $this->setTemplate('pimgento/launcher.phtml');

        $params = $this->getConfig()->getParams();
```
 
The code needs to be refactored to support the newly setup for uploading..

**Comment by a2xchip** [comment]
Thanks for fast reply. I will check how to fix it better may be dnd will move it faster to master that way.

**Comment by buttash** [comment]
I have written a script that you can call from the command line to perform ad hoc imports.  You'll need to modify if you do not use modman to pick up the Magento environment.  See https://github.com/buttash/PIMGento/tree/master/shell.

**Comment by a2xchip** [comment]
@buttash Thx! I already use it :)

**Comment by buttash** [comment]
@a2xchip - glad it works for you!

**Comment by a2xchip** [comment]
@buttash Like a charm :) but there are issues with scopable pim_catalog_multiselect/select attributes

**Comment by seblepers** [comment]
A pull request has been proposed : https://github.com/Agence-DnD/PIMGento/pull/177

**Comment by magentix** [comment]
Hi,

Now refer to the branch "magento-1.9.3". This branch fix compatiblity problem for Magento >= 1.9.3.
